### ISSUE TITLE
Issue sorting bug

### ISSUE THREAD

**Issue by benjaco** [k0shk0sh/FastHub#1728]
**FastHub Version: 4.6.7**  
**Android Version: 8.1.0 (SDK: 27)**  
**Device Information:**  
- **Manufacturer:** Google  
- **Model:** Pixel 2 XL  
---

 ![screenshot](https://i.imgur.com/6OMSpp4.png)

Error when selecting issue kind and milestone at the same time

**Comment by yakov116** [comment]
@benjaco Why do you have quotes around the milestone?

**Comment by benjaco** [comment]
Yes, I changed the query manually, didn't thought it was a limitation

**Comment by benjaco** [comment]
@yakov116 I don't know, it was put on automatically

**Comment by k0shk0sh** [comment]
Its not but you want to have it search for closed so don't remove it otherwise I'll list everything
